store/postgres/src/primary.rs

```diff
@@ -1116,7 +1131,13 @@ impl<'a> Connection<'a> {
             return Ok(site);
         }
 
-        self.create_site(shard, src.deployment.clone(), src.network.clone(), false)
+        self.create_site(
+            shard,
+            src.deployment.clone(),
+            src.network.clone(),
+            src.schema_version,
```

Comment:
This will be an important detail when we have more than one schema version: will copying create a deployment using the source's schema version or the latest version? My gut feeling is that it should always be the latest version, in which case `create_site` shouldn't take this as an argument and always use `CURRENT` for the new site.